android/assets/jsons/translations/French.properties

```diff
@@ -2568,6 +2568,7 @@ You returned captured units to us = Vous nous avez rendu des unités capturées
 
 Main Menu = Menu principal
 World Screen = Écran de jeu
+Auto Play Menu = Menu Jeu Auto
```

Comment:
Why is this "Auto Play" when everything else is AutoPlay? Let's be consistent
I know it's not your fault, but if you're already at it...

Reply:
We can probably just replace all English text and keep existing translations